api/backend.go

```diff
@@ -431,3 +443,58 @@ func appendIf(condition bool, services []gethnode.ServiceConstructor, service ge
 	}
 	return append(services, service)
 }
+
+func (b *StatusBackend) CreateContactCode() (string, error) {
+	selectedAccount, err := b.AccountManager().SelectedAccount()
+	if err != nil {
+		return "", err
+	}
+
+	st, err := b.statusNode.ShhExtService()
+	if err != nil {
+		return "", err
+	}
+
+	bundle, err := st.GetBundle(selectedAccount.AccountKey.PrivateKey)
+	if err != nil {
+		return "", err
+	}
+
+	base64Bundle, err := bundle.ToBase64()
+
+	return base64Bundle, err
+}
+
+func (b *StatusBackend) ProcessContactCode(contactCode string) error {
+	selectedAccount, err := b.AccountManager().SelectedAccount()
+	if selectedAccount == nil || err == account.ErrNoAccountSelected {
```

Comment:
Is this code correct for the case where `selectedAccount` is nil (what error do we want to return)?